src/webgpu/shader/execution/shader_io/compute_builtins.spec.ts

```diff
@@ -140,35 +138,16 @@ g.test('inputs')
       },
     });
 
-    // Helper to create a `size`-byte buffer with binding number `binding`.
-    function createBuffer(size: number, binding: number) {
-      const buffer = t.device.createBuffer({
-        size,
-        usage: GPUBufferUsage.STORAGE | GPUBufferUsage.COPY_SRC,
-      });
-      t.trackForCleanup(buffer);
-
-      bindGroupEntries.push({
-        binding,
-        resource: {
-          buffer,
-        },
-      });
-
-      return buffer;
-    }
-
     // Create the output buffers.
-    const bindGroupEntries: GPUBindGroupEntry[] = [];
-    const localIdBuffer = createBuffer(totalInvocations * 16, 0);
-    const localIndexBuffer = createBuffer(totalInvocations * 4, 1);
-    const globalIdBuffer = createBuffer(totalInvocations * 16, 2);
-    const groupIdBuffer = createBuffer(totalInvocations * 16, 3);
-    const numGroupsBuffer = createBuffer(totalInvocations * 16, 4);
+    const outputBuffer = t.device.createBuffer({
+      size: totalInvocations * (3 + 1 + 4 + 4 + 4) * 4,
```

Comment:
nit: I think it's better to use `size: totalInvocations * kOutputElementSize * 4` here? (need to move the offset const to above)

Reply:
Done